packages/grafana-ui/src/utils/tooltipUtils.ts

```diff
@@ -19,24 +21,25 @@ export function getPlacement(placement?: TooltipPlacement): Placement {
 }
 
 export function buildTooltipTheme(
-  theme: GrafanaTheme2,
+  theme: GrafanaTheme3,
   tooltipBg: string,
   toggletipBorder: string,
   tooltipText: string,
-  tooltipPadding: { topBottom: number; rightLeft: number }
+  tooltipPadding: { topBottom: keyof Spacing; rightLeft: keyof Spacing }
 ) {
   return {
     arrow: css({
       fill: tooltipBg,
     }),
     container: css({
       backgroundColor: tooltipBg,
-      borderRadius: theme.shape.radius.default,
+      borderRadius: theme.border.borderRadius.default,
       border: `1px solid ${toggletipBorder}`,
-      boxShadow: theme.shadows.z2,
+      boxShadow: theme.shadow.z2,
```

Comment:
I do not see the need for GrafanaTheme3 here, looks to me like we can keep using the old theme type (and add new tokens to it)